projects/packages/my-jetpack/_inc/components/connection-status-card/index.tsx

```diff
@@ -0,0 +1,395 @@
+import { Button, getRedirectUrl, H3, Text } from '@automattic/jetpack-components';
```

Comment:
This whole file was converted to TypeScript, so it's not as many changes as it seems